test/unit/engine_runtime.js

```diff
@@ -50,7 +50,7 @@ test('monitorStateDoesNotEqual', t => {
     });
     r.requestAddMonitor(prevMonitorState);
     r.requestUpdateMonitor(newMonitorDelta);
-    
+
```

Comment:
stray newline

Reply:
Nope: this was a blank line with 4 spaces on it, and now it's a blank line with no spaces on it :)